File: do_while.py
n = 0

while n <= 0:
    n = int(input('masukkan nilai n > 0 : '))

while n > 0:
    print('\nanak ayam turun ' + str(n))

    if n  == 1:
        break
    else:
        n = n - 1
    print('Mati satu tinggal ' + str(n))

print('Mati satu tinggal induknya\n')    

File: do_while.c
#include <stdio.h>
#include <stdlib.h>

int main()
{


    int n;

    do {
        printf("masukkan nilai N > 0 : ");
        scanf("%d", &n);        
    } while (n <= 0);

    do {
        printf("\nAnak Ayam Turun %d\n", n);
        if (n == 1) {
            break;
        } else {
            n = n - 1;
            printf("Mati satu tinggal %d \n", n);
        }
    } while (n > 0);
    
    printf("mati satu tinggal induknya \n");
    system("pause");
}